.github/workflows/reusable-mac-ci.yml

```
@@ -109,7 +104,7 @@ jobs:
         continue-on-error: false
         uses: eProsima/eProsima-CI/multiplatform/colcon_build@v0
         with:
-          colcon_meta_file: ${{ github.workspace }}/ci.meta
+          colcon_meta_file: ./src/fastrtps/.github/workflows/config/ci.meta
```

I think it's safer to keep abs paths everywhere. Applies to all changes
```suggestion
          colcon_meta_file: ${{ github.workspace }}/src/fastrtps/.github/workflows/config/ci.meta
```